libafl/Cargo.toml

```diff
@@ -17,7 +17,7 @@ all-features = true
 rustc-args = ["--cfg", "docsrs"]
 
 [features]
-default = ["std", "derive", "llmp_compression", "llmp_small_maps", "llmp_broker_timeouts", "rand_trait", "fork", "prelude", "gzip", "regex", "serdeany_autoreg", "tui_monitor", "libafl_bolts/xxh3"]
+default = ["std", "derive", "llmp_compression", "llmp_small_maps", "llmp_broker_timeouts", "rand_trait", "prelude", "gzip", "regex", "serdeany_autoreg", "tui_monitor", "libafl_bolts/xxh3"]
```

Comment:
Why not keep that default? Mostly people would want it

Reply:
it was initially changed for debugging purpose
i'll change it back